sys/net/network_layer/fib/fib.c

```diff
@@ -472,16 +482,29 @@ void fib_deinit(void)
     mutex_unlock(&mtx_access);
 }
 
-void fib_register_rrp(void)
+int fib_register_rrp(uint8_t *prefix, size_t prefix_size)
 {
     mutex_lock(&mtx_access);
 
+    if (notify_rrp_pos >= FIB_MAX_RRP) {
+        mutex_unlock(&mtx_access);
+        return -ENOMEM;
+    }
+
+    if ((prefix == NULL) || (prefix_size == 0)) {
```

Comment:
Makes sense. Other than the first comment, I have no other quibbles.. ACK when the 1st comment is addressed and travis is happy :)